lib/ios/RNNBridgeModule.m

```diff
@@ -218,6 +218,18 @@ - (instancetype)initWithCommandsHandler:(RNNCommandsHandler *)commandsHandler {
     });
 }
 
+RCT_EXPORT_METHOD(dismissAllOverlays
+                  : (NSString *)commandId componentId
+                  : (RCTPromiseResolveBlock)resolve rejecter
+                  : (RCTPromiseRejectBlock)reject) {
+    RCTExecuteOnMainQueue(^{
+      [self->_commandsHandler dismissAllOverlays:commandId
+                                      completion:^{
+                                        resolve(@(1));
```

Comment:
Keeping with `dismissAllModals` this should resolve to `nil`

Reply:
done 👍